docs/githubhooks/index.md

```diff
@@ -157,7 +157,10 @@ The IBM Bluemix DevOps Services hook replaces the RationalJazzHub hook. If you c
 2. Click **Webhooks & services**.
 ![GitHub web hooks and services link.][5]
 3. Click **Add service**, and from the **Available Services** list, select **IBM Bluemix DevOps Services**.
-4. If you are using public GitHub, type your IBM id and password. If you are using GitHub Enterprise, type your company user name and password. 
+4. If you are using public GitHub, type your federated ID and personal access token or IBM id and password. If you are using GitHub Enterprise, type your company user name and password.  

```

Comment:
@sktay Please see https://hub.jazz.net/ccm09/resource/itemName/com.ibm.team.workitem.WorkItem/59109 The blurb about GHE is incorrect and should be removed. The Note below looks good.. just need to fix the misspelling of "creating"...